src/summary/profile/identifiers-card.component.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import HorizontalLabelValue from "../cards/horizontal-label-value.component";
+import { match } from "react-router";
+import VerticalLabelValue from "../cards/vertical-label-value.component";
+
+export default function IdentifiersCard(props: IdentifiersCardProps) {
+  const labelStyles = {
+    color: "var(--omrs-color-ink-medium-contrast)"
+  };
+
+  const valueStyles = {
+    color: "var(--omrs-color-ink-high-contrast)",
+    fontWeight: 500
+  };
+
+  function isPreferred(identifier) {
```

Comment:
This function, as well as `preferredIdFirst`, could be pulled out of the IdentifiersCard function (maybe moved to the bottom of this whole file), since it's a pure function. There is a negligible, not important whatsoever, perf benefit to that. But I think that other advantage is just that it clarifies to maintainers that "this function doesn't rely on props or state."